presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -59,6 +59,46 @@ General Aggregate Functions
 
     Returns the geometric mean of all input values.
 
+.. function:: grouping(col1, ..., colN) -> integer
+
+    Returns a bit set converted to decimal, indicating which columns are present in a grouping. The function
+    must be used in conjunction with ``GROUPING SETS``, ``ROLLUP`` or ``CUBE`` and its arguments must
+    match exactly the columns referenced in the corresponding ``GROUPING SETS``, ``ROLLUP`` or ``CUBE``.
+
+    To compute the resulting bit set for a particular row, bits are assigned to the argument columns with
+    the rightmost column being the least significant bit. For a given grouping, a bit is set to 0 if the
```

Comment:
The spec is worded in a confusing way. At first I interpreted it the way you suggest but after trying out queries on Postgres and Teradata it turns out it's the other way around, i.e. as the doc suggests. Consider this query:
```sql
SELECT a, b, grouping(a, b)  
FROM (VALUES ('h', 'j', 11), ('k', 'l', 7)) AS t (a, b, c)  
GROUP BY GROUPING SETS ((a), (b));
```
The above will compute two `GROUP BY`s, one on column `a` and one on column `b`. When performing `GROUP BY a`, columns `b` and `c` are (using the spec terminology) "aggregated over". In other words, if we had aggregate functions they would operate on either column `b` or column `c` or both. Thus, when computing `GROUP BY a`, column `b` is aggregated over and so the resulting bit sequence is 01. During `GROUP BY b`, column `a` is aggregated over and the resulting bit sequence is 10. Thus the result is:
```
  a   |  b   | _col2 
------+------+-------
 h    | NULL |     1 
 k    | NULL |     1 
 NULL | j    |     2 
 NULL | l    |     2 
(4 rows)
```